src/utils/routes.ts

```diff
@@ -33,4 +34,12 @@ export const ROUTES: { [x: string]: Route } = {
     protected: true,
     index: false,
   },
+  undernames: {
+    text: 'Undernames',
+    icon: CubeIcon,
+    path: '/manage/ant/:id/undernames',
+    component: Undernames,
+    protected: true,
+    index: false,
+  },
```

Comment:
these routes are really only used for the nav menu, so not needed here

Reply:
removed